southworks/botframework-solutions
# [TypeScript][Samples/Template] Update the SDK dependencies

## Purpose
_What is the context of this pull request? Why is it being done?_
Update the SDK dependencies version for the Samples and Templates

## Changes
_Are there any changes that need to be called out as significant or particularly difficult to grasp?_ (Include illustrative screenshots for context if applicable.)

We updated the following files in `skill/assistant templates` & `samples assistant/skill`:
- `package.json` 
- `package-lock.json`
	
![image](https://user-images.githubusercontent.com/40918752/63623916-52d46a00-c5d1-11e9-866a-8c903aad9f0a.png)


## Tests
Is this covered by existing tests or new ones? If no, why not?
Yes, it's covered by existing tests.

## Testing Steps
1. Go to `.\templates\Virtual-Assistant-Template\typescript\samples\sample-assistant` open a terminal and execute `pwsh.exe -File deployment\scripts\deploy.ps1`.
1. Execute `npm install` to install the dependencies.
1. Execute `npm run build`.
1. Execute `npm run test` check that the test are passing successfully.
1. Execute `npm run start`.
1. Verify that the behavior of the bot hasn't changed having led to the latest version of the libraries.

## Feature Plan
_Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues._

Continued updating of the dependencies in `BotBuilder-Skills` and `BotBuilder-Solutions`

### Checklist
#### General

#### Bots

#### Deployment Scripts
